addons/stock_account/models/product.py

```diff
@@ -20,6 +20,12 @@ class ProductTemplate(models.Model):
         help="If checked, the valuation will be specific by Lot/Serial number.",
     )
 
+    @api.onchange('tracking')
+    def _onchange_tracking(self):
+        if self.tracking == "none" and self.lot_valuated:
+            self.lot_valuated = False
+        super()._onchange_tracking()
```

Comment:
Can you move the `super` at the beginning ?
So we ensure `tracking` doesn't change